src/Groups/sub.jl

```diff
@@ -493,14 +493,14 @@ See also [`lower_central_series`](@ref) and [`nilpotency_class`](@ref).
 ```jldoctest
 julia> upper_central_series(dihedral_group(8))
 3-element Vector{SubPcGroup}:
- Sub-pc group of order 8
- Sub-pc group of order 2
- Sub-pc group of order 1
+ Sub-pc group of order 8 with 3 generators f3, f1, f2
+ Sub-pc group of order 2 with 1 generator f3
+ Sub-pc group of order 1 with 0 generators
```

Comment:
Not a remark about this pull request but about GAP:

Apparently `LowerCentralSeries` and `UpperCentralSeries` behave differently for the dihedral group of order 8, the trivial subgroup has either one or zero generators, and the full group is either the original group or a new group with permuted generators.

The element `<identity> of ...` looks strange already in GAP, but even more in Oscar.